EnglishFilter/sections/general_extensions.txt

```diff
@@ -2321,7 +2323,8 @@ pogo.com#%#AG_defineProperty('pogo.intermission.staticAdIntermissionPeriod', {va
 ! https://github.com/AdguardTeam/AdguardFilters/issues/28680
 ! https://github.com/AdguardTeam/AdguardFilters/issues/28441
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27743
-oladblock.me,oladblock.xyz,oladblock.services,oload.space,oload.live,oload.club,openload.pw,oload.fun#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });
+ovagames.com - popups
+ovagames.com,oladblock.me,oladblock.xyz,oladblock.services,oload.space,oload.live,oload.club,openload.pw,oload.fun#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });
```

Comment:
`ovagames.com - popups` without `!`.
What's the point of adding the domain here `#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });`?

It seems only `window.open = function() {};` is useful here, the others are not.
So, the popup is fixed by `||zxclan.com^$third-party`, but you can also add `ovagames.com#%#//scriptlet("set-constant", "window.open", "noopFunc")`.